### PR TITLE
ref: Refactor event metrics to distinguish between relay vs non-relay

### PR DESCRIPTION
**Pull request by untitaker**

### CODE REVIEW

File: src/sentry/event_manager.py [hunk 1]
```diff
@@ -810,9 +808,15 @@ def save(self, project_id, raw=False, assume_normalized=False):
             skip_consume=raw,
         )
 
-        metrics.timing("events.latency", received_timestamp - recorded_timestamp)
+        metric_tags = {"from_relay": "_relay_processed" in self._data}
 
-        metrics.timing("events.size.data.post_save", event.size)
+        metrics.timing("events.latency", received_timestamp - recorded_timestamp, tags=metric_tags)
+        metrics.timing("events.size.data.post_save", event.size, tags=metric_tags)
+        metrics.timing(
```
Comment (mitsuhiko) [inline src/sentry/event_manager.py]: This should be an incr

**Review by mitsuhiko — DISMISSED** [review]

File: src/sentry/event_manager.py (hunk 1 above)
Comment (untitaker) [reply src/sentry/event_manager.py]: Then we can only see errors/time-unit while this can give us avg/min/max value

Comment (mitsuhiko) [reply src/sentry/event_manager.py]: the one i want is "does sum of relay/non relay path" diverge. So i think we need the incr.

Comment (untitaker) [reply src/sentry/event_manager.py]: nvm, you're right. max/min/avg of incr is only useless when you incr by 1 everytime

**Review by jan-auer — APPROVED** [review]

**Pull request merged**